### PR TITLE
Feature/configuracion ixe

### PR DESCRIPTION
**Pull request by AdIx42**
## ¿Cuál es el cambio?
<Aquí se describe el cambio>

## Tipo de cambio

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
[ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] I have added necessary documentation (if appropriate)
- [x] I have added necessary tags for QA automation (if appropriate)
- [x] I did review existing Pull Requests before submit

### CODE REVIEW

File: src/main/java/com/example/HelloController.java [hunk 1]
```diff
@@ -10,4 +10,9 @@ public class HelloController {
     public String hello() {
         return "Hello, Worldx!";
     }
+    
+    @Get("/Ixe")
+    public String helloIxe() {
```
Comment (jjsainfo) [inline src/main/java/com/example/HelloController.java:15]: buen metodo Ixe

File: src/test/java/com/example/HelloControllerTest.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    @Test
+    void testHelloEndpoint() {
+        HttpRequest<String> request = HttpRequest.GET("/hello");
+        HttpResponse<String> response = client.toBlocking().exchange(request, String.class);
+
+        assertEquals("Hello, Worldx!", response.body());
+    }
+    
+    @Test
+    void testHelloEndpointIxe() {
```
Comment (jjsainfo) [inline src/test/java/com/example/HelloControllerTest.java:29]: A mi parecer este metodo podria quedar en un test aparte por la funcionalidad que tiene

**Review by jjsainfo — APPROVED** [review]
vientos casi todo ok :p

File: src/main/java/com/example/HelloController.java (hunk 1 above)
Comment (neucast) [reply src/main/java/com/example/HelloController.java:15]: Method does not return a string. :(

File: src/test/java/com/example/HelloControllerTest.java (hunk 2 above)
Comment (neucast) [reply src/test/java/com/example/HelloControllerTest.java:29]: I think it is better that the test must be written in and specific test class.

**Pull request merged**